```
01:04:52  /home/build-user/llvm-project/lldb/source/ValueObject/DILParser.cpp:54:40: error: no member named 'GetFunctionName' in 'std::shared_ptr<lldb_private::StackFrame>'; did you mean to use '->' instead of '.'?
01:04:52                                      ctx.GetFunctionName());
01:04:52                                         ^
01:04:52                                         ->
01:04:52  /home/build-user/llvm-project/lldb/source/ValueObject/DILParser.cpp:53:18: error: no member named 'createStringErrorV' in namespace 'llvm'
01:04:52      return llvm::createStringErrorV("no compile unit for frame: {}",
01:04:52             ~~~~~~^
01:04:52  2 errors generated.
```

--- a/lldb/source/ValueObject/DILParser.cpp
+++ b/lldb/source/ValueObject/DILParser.cpp
@@ -50,8 +50,8 @@ GetTypeSystemFromCU(std::shared_ptr<StackFrame> ctx) {
   SymbolContext symbol_context =
       ctx->GetSymbolContext(lldb::eSymbolContextCompUnit);
   if (!symbol_context.comp_unit)
-    return llvm::createStringErrorV("no compile unit for frame: {}",
-                                    ctx.GetFunctionName());
+    return llvm::createStringError(llvm::formatv("no compile unit for frame: {}",
+                                                 ctx->GetFunctionName()));
 
   lldb::LanguageType language = symbol_context.comp_unit->GetLanguage();
 

